src/Gui/3Dconnexion/GuiNativeEventWin32.cpp

```diff
@@ -1071,6 +1074,12 @@ bool Gui::GuiNativeEvent::TranslateSpaceMouseOldGeneric(UINT nInputCode, PRAWINP
     bool processed = false;
     bool bIsForeground = (nInputCode == RIM_INPUT);
 
+    // Initialize with some random bit of data before we look up the real value later on
```

Comment:
Cannot judge on this because for me the compilation of PreCompiled.cpp causes a failure due to boost and thus cannot continue with build FreeCADGui.